### ISSUE TITLE
Implement group administration

### ISSUE THREAD

**Issue by magpern** [yllibed/Zigbee2MqttAssistant#244]
The administration of groups, adding removing devices etc is quite a task. Not for the beginner. A GUI for this would help alot.

All information is here

https://github.com/Koenkk/zigbee2mqtt/issues/3298

**Comment by jd1900** [comment]
+1

I miss this feature a lot. 
Currently, when we have a group, this is the given information, which is a bit misleading:

NO COMPONENTS Maybe this device is not (yet) supported by Zigbee2Mqtt?
--

**Comment by carldebilly** [comment]
I think the first step would be to detect groups and remove this useless message :-)

**Comment by magpern** [comment]
@carldebilly yes, but I think Z2MA needs to change its way of getting the device information first. Instead of reading a mix of zigbee2mqtt and homeassistant topics, it should ask zigbee2mqtt for all devices, and from this data build a device structure. This way one could more easily separate devices from groups, and further on, more easy work with groups.
Also, it would probably make the transition to zigbee2mqtt upcoming new MQTT structure

**Comment by carldebilly** [comment]
I agree. That's why there's a pending PR on Z2M for getting "components".

**Comment by itavero** [comment]
I reckon you mean koenkk/zigbee2mqtt#2692 ?
It is currently marked as _stale_.

**Comment by carldebilly** [comment]
Yes it is because I'm way too busy with a whole family at home right now, sorry. Maybe in few weeks I'll have more time available to continue ongoing projects, sorry.

**Comment by epicurean70** [comment]
@carldebilly thank you for your time and effort to make this incredibly useful app. I think all of us openhab users are waiting eagerly for the "no components" error to be resolved. BUT, family is always the most important. Keep safe and take care!
